src/frontend/src/components/Map.js

```diff
@@ -4,11 +4,23 @@ import {
   TileLayer,
   LayersControl,
   Polygon,
+  FeatureGroup,
   useMap,
   useMapEvents,
 } from "react-leaflet";
+import { EditControl } from "react-leaflet-draw";
 import "leaflet/dist/leaflet.css";
+import "leaflet-draw/dist/leaflet.draw.css";
 import * as h3 from "h3-js/legacy";
+import L from "leaflet";
+
+// Customizing the tooltip messages
+L.drawLocal.draw.toolbar.buttons.rectangle = 'REMOVE annotation hexagons';
```

Comment:
just an idea if it's not too difficult and I think it might clear confusions while using. We store 'presence' or 'absence' whiever is selected in app.js, can we prop those into this component, so that if 'presence' it will say 'REMOVE presence hexagons' instead of just annotation? if it's too much work whatever we have is fine too it might need clear instructions